### DISCUSSION TITLE
Can projections-ios can use with SwiftUI Projects

### DISCUSSION THREAD

**Discussion by thothai94** [ngageoint/projections-ios#2] [Q&A] [answered]
Hello.
I am using SwiftUI for current iOS project and I would like to use this library. Please tell me Can I use projections-ios and how to import to my project?
Thanks alot!

**Comment by bosborn** [comment, marked as answer]
Should be able to.  [Include the library](https://github.com/ngageoint/projections-ios#include-library) and call [from Swift](https://github.com/ngageoint/projections-ios#swift).
If you also want simple feature geometries, use [simple-features-proj-ios](https://github.com/ngageoint/simple-features-proj-ios).

**Reply by thothai94** [reply]
Does this Project has plan to support xcFramework or release on Swift Package Manager (SPM)?

**Reply by bosborn** [reply]
Not opposed but no immediate plans.  I've been able to get some of our Objective-C libraries building & testing with SPM, but not in a way where dependent libraries can build using them.
